### ISSUE TITLE
Crash while using 'Drone Painter'

### ISSUE THREAD

**Issue by CKTheUltimate** [WilliamEz/CustomDrones#5]
If I click the 'apply colour' or 'apply palette' button when using the 'Drone Painter' item,the game suddenly crashes after displaying 'shutting down internal server...' .
Here is the crash report:
---- Minecraft Crash Report ----

WARNING: coremods are present:
Contact their authors BEFORE contacting forge

// Don't do that.

Time: 8/4/16 3:41 PM
Description: Updating screen events

java.lang.NoClassDefFoundError: scala/actors/threadpool/Arrays
    at williamle.drones.network.server.PacketDronePaint.<init>(PacketDronePaint.java:31)
    at williamle.drones.gui.GuiPainter.func_146284_a(GuiPainter.java:122)
    at net.minecraft.client.gui.GuiScreen.func_73864_a(GuiScreen.java:458)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java:320)
    at williamle.drones.gui.util.GuiContainerPanel.func_73864_a(GuiContainerPanel.java:30)
    at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:546)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:515)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1700)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1055)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.ClassNotFoundException: scala.actors.threadpool.Arrays
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 17 more
Caused by: java.lang.NullPointerException
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
    ... 19 more
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Thread: Client thread
Stacktrace:
    at williamle.drones.network.server.PacketDronePaint.<init>(PacketDronePaint.java:31)
    at williamle.drones.gui.GuiPainter.func_146284_a(GuiPainter.java:122)
    at net.minecraft.client.gui.GuiScreen.func_73864_a(GuiScreen.java:458)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java:320)
    at williamle.drones.gui.util.GuiContainerPanel.func_73864_a(GuiContainerPanel.java:30)
    at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:546)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:515)

-- Affected screen --
Details:
    Screen name: williamle.drones.gui.GuiPainter

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['CKTheUltimate'/200, l='MpServer', x=-599.79, y=9.00, z=-728.39]]
    Chunk stats: MultiplayerChunkCache: 1089, 1089
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: World: (-597,4,-720), Chunk: (at 11,0,0 in -38,-45; contains blocks -608,0,-720 to -593,255,-705), Region: (-2,-2; contains chunks -64,-64 to -33,-33, blocks -1024,0,-1024 to -513,255,-513)
    Level time: 95224 game time, 30583 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 103 total; [EntityPlayerSP['CKTheUltimate'/200, l='MpServer', x=-599.79, y=9.00, z=-728.39], EntityChicken['Chicken'/41, l='MpServer', x=-679.50, y=4.00, z=-791.50], EntityVillager['Villager'/42, l='MpServer', x=-676.70, y=5.00, z=-753.70], EntityVillager['Villager'/43, l='MpServer', x=-674.30, y=5.00, z=-753.70], EntityChicken['Chicken'/45, l='MpServer', x=-675.11, y=4.00, z=-718.26], EntityPig['Pig'/46, l='MpServer', x=-675.51, y=4.00, z=-717.15], EntityChicken['Chicken'/48, l='MpServer', x=-673.80, y=4.00, z=-662.10], EntityChicken['Chicken'/49, l='MpServer', x=-674.50, y=4.00, z=-666.50], EntityChicken['Chicken'/50, l='MpServer', x=-676.51, y=4.00, z=-669.47], EntityChicken['Chicken'/51, l='MpServer', x=-675.50, y=4.00, z=-665.50], EntitySheep['Sheep'/54, l='MpServer', x=-676.64, y=4.00, z=-665.83], EntityHorse['Horse'/55, l='MpServer', x=-664.90, y=5.00, z=-759.98], EntityVillager['Villager'/56, l='MpServer', x=-672.25, y=4.00, z=-747.64], EntityVillager['Villager'/57, l='MpServer', x=-662.73, y=5.00, z=-760.56], EntitySheep['Sheep'/58, l='MpServer', x=-667.27, y=4.00, z=-740.70], EntityHorse['Horse'/59, l='MpServer', x=-670.17, y=4.00, z=-744.84], EntitySlime['Slime'/60, l='MpServer', x=-663.58, y=6.17, z=-743.63], EntitySheep['Sheep'/61, l='MpServer', x=-663.40, y=4.00, z=-736.25], EntitySheep['Sheep'/62, l='MpServer', x=-652.83, y=4.00, z=-729.78], EntityChicken['Chicken'/63, l='MpServer', x=-671.94, y=4.00, z=-716.81], EntitySlime['Slime'/64, l='MpServer', x=-664.32, y=4.00, z=-683.38], EntitySlime['Slime'/66, l='MpServer', x=-653.83, y=5.00, z=-795.15], EntitySheep['Sheep'/67, l='MpServer', x=-640.64, y=4.00, z=-743.35], EntitySlime['Slime'/68, l='MpServer', x=-645.42, y=4.33, z=-749.04], EntitySlime['Slime'/10308, l='MpServer', x=-640.27, y=4.00, z=-705.36], EntityPig['Pig'/69, l='MpServer', x=-653.61, y=4.00, z=-726.74], EntitySlime['Slime'/71, l='MpServer', x=-639.10, y=4.53, z=-790.92], EntitySlime['Slime'/72, l='MpServer', x=-651.96, y=5.17, z=-741.39], EntitySlime['Slime'/73, l='MpServer', x=-635.48, y=4.33, z=-727.66], EntitySlime['Slime'/74, l='MpServer', x=-627.39, y=4.00, z=-700.55], EntitySlime['Slime'/75, l='MpServer', x=-631.91, y=4.53, z=-666.78], EntitySlime['Slime'/76, l='MpServer', x=-636.43, y=4.00, z=-665.13], EntitySlime['Slime'/83, l='MpServer', x=-621.76, y=4.00, z=-792.36], EntitySlime['Slime'/84, l='MpServer', x=-612.05, y=4.95, z=-797.92], EntitySlime['Slime'/85, l='MpServer', x=-605.93, y=5.08, z=-787.71], EntitySlime['Slime'/86, l='MpServer', x=-610.74, y=4.00, z=-758.25], EntitySlime['Slime'/87, l='MpServer', x=-608.23, y=4.67, z=-750.32], EntityCreeper['Creeper'/88, l='MpServer', x=-619.20, y=4.00, z=-717.55], EntitySlime['Slime'/89, l='MpServer', x=-617.62, y=4.92, z=-708.05], EntitySlime['Slime'/90, l='MpServer', x=-616.30, y=4.72, z=-694.04], EntitySlime['Slime'/100, l='MpServer', x=-601.84, y=4.95, z=-795.82], EntitySlime['Slime'/101, l='MpServer', x=-586.53, y=4.92, z=-772.27], EntitySlime['Slime'/102, l='MpServer', x=-598.38, y=4.00, z=-750.88], EntitySlime['Slime'/103, l='MpServer', x=-608.24, y=4.67, z=-752.53], EntitySlime['Slime'/104, l='MpServer', x=-601.48, y=4.00, z=-739.00], EntityCreeper['Creeper'/105, l='MpServer', x=-591.50, y=4.00, z=-729.79], EntityDrone['Drone'/106, l='MpServer', x=-606.50, y=6.00, z=-733.50], EntityDrone['Drone'/107, l='MpServer', x=-601.50, y=9.00, z=-728.50], EntityCreeper['Creeper'/108, l='MpServer', x=-600.72, y=4.00, z=-735.42], EntitySlime['Slime'/109, l='MpServer', x=-605.09, y=4.72, z=-741.05], EntitySlime['Slime'/110, l='MpServer', x=-595.09, y=4.00, z=-720.98], EntitySheep['Sheep'/111, l='MpServer', x=-604.58, y=4.00, z=-672.41], EntitySheep['Sheep'/112, l='MpServer', x=-595.82, y=4.00, z=-682.63], EntityPig['Pig'/113, l='MpServer', x=-593.06, y=4.00, z=-680.93], EntitySlime['Slime'/114, l='MpServer', x=-609.92, y=4.00, z=-691.64], EntityChicken['Chicken'/115, l='MpServer', x=-607.19, y=4.00, z=-663.65], EntityChicken['Chicken'/116, l='MpServer', x=-602.18, y=4.00, z=-668.91], EntityItem['item.item.egg'/117, l='MpServer', x=-601.84, y=4.00, z=-668.85], EntitySheep['Sheep'/120, l='MpServer', x=-595.13, y=4.00, z=-649.79], EntitySheep['Sheep'/121, l='MpServer', x=-596.77, y=4.00, z=-653.39], EntityPig['Pig'/122, l='MpServer', x=-592.21, y=4.00, z=-648.56], EntityCreeper['Creeper'/125, l='MpServer', x=-588.68, y=4.00, z=-742.17], EntitySlime['Slime'/126, l='MpServer', x=-578.81, y=4.95, z=-729.23], EntityChicken['Chicken'/127, l='MpServer', x=-587.82, y=4.00, z=-678.41], EntityPig['Pig'/128, l='MpServer', x=-584.64, y=4.00, z=-678.15], EntityItem['item.item.egg'/129, l='MpServer', x=-591.49, y=4.00, z=-676.69], EntityChicken['Chicken'/130, l='MpServer', x=-591.32, y=4.00, z=-669.53], EntityPig['Pig'/131, l='MpServer', x=-579.67, y=4.00, z=-661.97], EntityItem['item.item.egg'/132, l='MpServer', x=-591.01, y=4.00, z=-669.61], EntitySlime['Slime'/133, l='MpServer', x=-590.47, y=4.67, z=-671.43], EntityPig['Pig'/136, l='MpServer', x=-587.93, y=4.00, z=-648.65], EntitySlime['Slime'/138, l='MpServer', x=-579.55, y=5.08, z=-805.68], EntitySlime['Slime'/139, l='MpServer', x=-573.25, y=4.00, z=-778.82], EntitySlime['Slime'/140, l='MpServer', x=-566.70, y=4.00, z=-770.01], EntitySheep['Sheep'/141, l='MpServer', x=-568.19, y=4.00, z=-705.40], EntitySlime['Slime'/142, l='MpServer', x=-544.35, y=4.00, z=-719.02], EntitySlime['Slime'/143, l='MpServer', x=-577.89, y=4.53, z=-707.33], EntityHorse['Horse'/144, l='MpServer', x=-564.85, y=4.00, z=-696.47], EntityHorse['Horse'/145, l='MpServer', x=-563.85, y=4.00, z=-694.87], EntitySlime['Slime'/146, l='MpServer', x=-567.47, y=4.00, z=-692.01], EntitySlime['Slime'/147, l='MpServer', x=-564.38, y=4.08, z=-678.37], EntitySlime['Slime'/151, l='MpServer', x=-549.49, y=4.08, z=-763.92], EntitySlime['Slime'/152, l='MpServer', x=-543.58, y=4.00, z=-753.55], EntitySheep['Sheep'/153, l='MpServer', x=-549.37, y=4.00, z=-747.40], EntitySheep['Sheep'/154, l='MpServer', x=-544.77, y=4.00, z=-745.44], EntityChicken['Chicken'/155, l='MpServer', x=-552.54, y=4.00, z=-737.11], EntitySheep['Sheep'/156, l='MpServer', x=-555.60, y=4.00, z=-710.49], EntitySheep['Sheep'/157, l='MpServer', x=-555.24, y=4.00, z=-708.68], EntitySheep['Sheep'/158, l='MpServer', x=-556.72, y=4.00, z=-710.82], EntityHorse['Horse'/159, l='MpServer', x=-558.11, y=4.00, z=-700.73], EntitySheep['Sheep'/160, l='MpServer', x=-550.64, y=4.00, z=-703.97], EntitySheep['Sheep'/161, l='MpServer', x=-552.76, y=4.00, z=-664.40], EntitySheep['Sheep'/162, l='MpServer', x=-548.04, y=4.00, z=-668.24], EntitySheep['Sheep'/163, l='MpServer', x=-554.58, y=4.00, z=-664.39], EntityPig['Pig'/164, l='MpServer', x=-550.56, y=4.00, z=-667.21], EntityPig['Pig'/165, l='MpServer', x=-546.49, y=4.00, z=-656.52], EntityPig['Pig'/166, l='MpServer', x=-550.54, y=4.00, z=-661.39], EntitySheep['Sheep'/167, l='MpServer', x=-556.22, y=4.00, z=-657.39], EntitySheep['Sheep'/168, l='MpServer', x=-554.42, y=4.00, z=-655.14], EntityPig['Pig'/176, l='MpServer', x=-530.95, y=4.00, z=-805.90], EntitySheep['Sheep'/180, l='MpServer', x=-529.50, y=4.00, z=-804.50], EntityPig['Pig'/181, l='MpServer', x=-519.77, y=4.00, z=-804.47], EntitySheep['Sheep'/182, l='MpServer', x=-528.41, y=4.00, z=-802.27]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:496)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2660)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:392)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.10.2
    Operating System: Mac OS X (x86_64) version 10.11.3
    Java Version: 1.8.0_74, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 327402904 bytes (312 MB) / 693161984 bytes (661 MB) up to 4281597952 bytes (4083 MB)
    JVM Flags: 5 total; -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP 9.32 Powered by Forge 12.18.1.2045 Optifine OptiFine_1.10.2_HD_U_C2 4 mods loaded, 4 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.19} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader](forge-1.10.2-12.18.1.2045.jar) 
    UCHIJAAAA   Forge{12.18.1.2045} [Minecraft Forge](forge-1.10.2-12.18.1.2045.jar) 
    UCHIJAAAA   drones{1.10.2-1.4.0} [CustomDrones](CustomDrones-1.4.0-mc1.10.2-2.jar) 
    Loaded coremods (and transformers): 
    Launched Version: 1.10.2-forge1.10.2-12.18.1.2045
    LWJGL: 2.9.2
    OpenGL: Intel(R) Iris(TM) Graphics 6100 GL version 2.1 INTEL-10.12.13, Intel Inc.
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

```
Using VBOs: Yes
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: 
Current Language: English (United Kingdom)
Profiler Position: N/A (disabled)
CPU: 4x Intel(R) Core(TM) i5-5287U CPU @ 2.90GHz
OptiFine Version: OptiFine_1.10.2_HD_U_C2
Render Distance Chunks: 16
Mipmaps: 4
Anisotropic Filtering: 1
Antialiasing: 0
Multitexture: false
Shaders: null
OpenGlVersion: 2.1 INTEL-10.12.13
OpenGlRenderer: Intel(R) Iris(TM) Graphics 6100
OpenGlVendor: Intel Inc.
CpuCount: 4
```

**Comment by WilliamEz** [comment]
This will be fixed in the next update
Thanks for the report :D

**Comment by WilliamEz** [comment]
Fixed in 1.5.0
